sympy/ntheory/digits.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import print_function, division
+
+import re
+
+from collections import Counter
+
+
+def is_pandigital(n, zeroless=False, freq='1+'):
```

Comment:
I've thought of replacing the keyword `zeroless` to `digit_set={'1', '2', '3'}` to make it more generalized.

Reply:
A zeroless pandigital number is one without a 0 digit, and term zeroless (and its counterpart zerofull) are known terms with respect to pandigital numbers.

http://mathworld.wolfram.com/PandigitalNumber.html